### ISSUE TITLE
Change repository name

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/bio_mobilization_workshop#4]
I propose we change this repository name to `bio_mobilization_workshop`.

Oceanhackweek had a nice approach of using the active `gh-pages` branch for the current year. Once the year is complete, move the content to a new branch that indicates the year (eg. `2022_site`). Then work on/update the `gh-pages` branch for the next iteration of the workshop. 

We can do something similar for `main` as well to preserve the datasets/code that were worked on that year. Give the branch a name like `2022_main`. 

I would like this to be somewhat futureproof in that we can use it for other workshops we might want to host in the future.

**Comment by albenson-usgs** [comment]
I like this. Are there any negative consequences to changing the repo name that I'm not thinking of?

**Comment by 7yl4r** [comment]
Yep, that new name is a huge improvement. I also love the branch organization approach described.

**Comment by MathewBiddle** [comment]
changed to `bio_mobilization_workshop`. I think everything else sorted itself out. open a new ticket if you see something wonky.
